### DISCUSSION TITLE
importing fluid libraries on MATLAB

### DISCUSSION THREAD

**Discussion by alexb3-svg** [CoolProp/CoolProp#2233] [General]
Hello,
I am trying to build a cascade refrigeration system on MATLAB/Simscape exploiting R134a and R452a. In order to do that I need the fluids' properties to put inside the "Two-Phase Fluid Properties (2P)" block of Simscape. The Matlab version is r2022b and coolprop version is 6.4.3. When it comes to import R134a properties everything is ok, but regarding R452a an appropriate mixture should be created (11% R32, 59% R125 and 30% R1234yf). I tried to create it with the help of online guides but couldn't do it.  Another easier option could be using R404A (not a mixture, already available in coolprop's libraries), but when I try to run the same code written for R134A errors show up. Does anybody know how to fix it?

**Comment by ibell** [comment]
Can you please show what you tried to do to initialize the mixture?

**Reply by ibell** [reply]
And what errors you obtained?
